dashboard/app/models/unit.rb

```diff
@@ -928,7 +929,7 @@ def csa?
   end
 
   def csc?
-    Unit.unit_in_category?('csc', name)
+    in_initiative?('CSC')
```

Comment:
verified all course offerings in this category which have published versions already have the correct marketing initiative